Python/symtable.c

```diff
@@ -360,8 +460,12 @@ _PySymtable_Build(mod_ty mod, PyObject *filename, PyFutureFeatures *future)
         return NULL;
     }
     /* Make the second symbol analysis pass */
-    if (symtable_analyze(st))
+    if (symtable_analyze(st)) {
+#if _PY_DUMP_SYMTABLE
+        dump_symtable(st->st_top);
+#endif
```

Comment:
I didn't add this because I thought it might often be too noisy? But I agree it's nice to make this easy; people can always add more conditions while debugging if the output is overwhelming.